cdk/lib/constructs/api.ts

```diff
@@ -171,15 +176,14 @@ export class Api extends Construct {
     props.usageAnalysis?.ddbBucket.grantRead(handlerRole);
     props.largeMessageBucket.grantReadWrite(handlerRole);
 
-    const handler = new DockerImageFunction(this, "Handler", {
-      code: DockerImageCode.fromImageAsset(
-        path.join(__dirname, "../../../backend"),
-        {
-          platform: Platform.LINUX_AMD64,
-          file: "Dockerfile",
-          exclude: [...excludeDockerImage],
-        }
-      ),
+    const handler = new PythonFunction(this, "HandlerV2", {
```

Comment:
renamed the construct id (`HandlerV2`) to re-create the handler. It is required because we cannot change package type (docker to zip) of an existing function.